# no-inline-styles

## Rule Details

This rule disables the usage of inline styles.

## Examples

Examples of **incorrect** code for this rule:

```jsx
<div style={{ display: "block" }}></div>
```

Examples of **correct** code for this rule:

```jsx
<div>Safe Content</div>
```
